package.json

```diff
@@ -15,8 +15,10 @@
     "actsAs": [
       "handler"
     ],
+    "stripes.type": "app",
```

Comment:
I don't think you want the `stripes.` prefix here; that's implied by being a property of the `stripes` object starting on line 14.

Reply:
Also, I saw @mkuklis [add support for `type: "handler"`](https://github.com/folio-org/stripes-cli/pull/231) in stripes-cli so we can use that here, right?